lib/europeana/i18n.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
+require 'i18n-js'
 require 'europeana/i18n/engine' if defined?(Rails)
+require 'europeana/i18n/railtie' if defined? Rails
```

Comment:
Let's enclose both of these `require` statements in one `if defined?(Rails)` block.